core/util/instagram.py

```diff
@@ -0,0 +1,250 @@
+"""
+OWASP Maryam!
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
```

Comment:
I feel instagram is already a module listed in smn's to do list.
Please get an Issue approved by @saeeddhqan before adding on modules.
Helps in saving conflicts

Reply:
No problem. He used api